hologram_dashboard/extension/src/proxy.conf.json

```diff
@@ -0,0 +1,7 @@
+{
+  "/": {
+      "target": "http://localhost:8000",
+      "secure": false,
+      "logLevel": "debug"
+  }
+}
```

Comment:
can I know what this file is for?

Reply:
It's because without it we cannot communicate with the backend since they have two different base address. We will get the cors error access-control-allow-origin without it.